tests/units/iop/test_iop_process_file.py

```diff
@@ -0,0 +1,40 @@
+import xml.etree.ElementTree as ET
+
+from iop.iop_process_file import enhance_iop, enrich_iop, iop_validate_record
+from iop.parser import IOPParser
+from pytest import fixture
+
+
+@fixture
+def article(shared_datadir):
+    file_content = (shared_datadir / "example1.xml").read_text()
+    return ET.fromstring(file_content)
+
```

Comment:
in general for fixtures is better to use `yield` https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#yield-fixtures-recommended